.scrutinizer.yml

```diff
@@ -0,0 +1,2 @@
+language: php
```

Comment:
I guess we can not add this. Or is there any benefit from it? I considered scrutinizer obsolete for some year now

Reply:
I used to use it for the "code quality" insight + badge, but there's no real benefit actually. Will remove it.